python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -395,7 +537,8 @@ def prepareDcrInputs(self, templateCoadd, warpRefList, weightList):
         self.log.info("Selected parallactic angles:\n%s", angleDict)
         self.log.info("Selected PSF sizes:\n%s", psfSizeDict)
         self.bufferSize = int(np.ceil(np.max(dcrShifts)) + 1)
-        psf = self.selectCoaddPsf(templateCoadd, warpRefList)
+#        psf = self.selectCoaddPsf(templateCoadd, warpRefList)
```

Comment:
remove comment

Reply:
This was an accidental change that made it in from an unrelated test. I'll revert this line.